### DISCUSSION TITLE
Correct the voice for a reader?

### DISCUSSION THREAD

**Discussion by bmcgonag** [DrewThomasson/VoxNovel#22] [General]
First and foremost, thank you so much for making this! It's absolutely awesome!

When it pulls the information for the book, and shows the predetermined voices for the characters, do I just type in the right character if it got them messed up?  I've tried that, but noticed when I hit "Load Book" it resets everything I corrected.   

Will it just take what I've changed when I click "Generate Audio"?

Thanks again,

**Comment by DrewThomasson** [comment]
SO SORRY ABOUT THE LATE RESPONSE!!

Weird,... you should just be able to select the voices you want for each character and itll use the selections/ modifications when generating the audiobook?

**Comment by DrewThomasson** [comment]
Your using the gui version correct?

**Comment by DrewThomasson** [comment]
For anyone arriving to this discussion:


## Status Update:

**Now added and supported in VoxNovel**

https://github.com/DrewThomasson/VoxNovel/issues/29#issuecomment-2323501113
